src/presentation/assets/LinkIcon.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+
+export default function UploadIcon() {
```

Comment:
Faltou renomear a função para LinkIcon

```suggestion
export default function LinkIcon() {
```